src/chunks/vocab/subword.rs

```diff
@@ -173,25 +214,30 @@ where
 pub trait SubwordIndices {
     /// Return the subword indices of the subwords of a string,
     /// according to the subword vocabulary.
-    fn subword_indices(&self, word: &str) -> Option<Vec<usize>>;
+    fn subword_indices(&self, word: &str, hash_word: bool) -> Option<Vec<usize>>;
```

Comment:
While looking at the trait definition, the `bool` param is fine, but at calling sites it's not really self-explanatory what `true` or `false` as the second argument to this function means. Could be nicer and more explicit to use an enum? Although I don't have a strong opinion on that.

Still, a short explanation of the expected behaviour given this param in the docs would be great!

Reply:
Using an enum here is a great idea!

Reply:
I added the following enum:

```
pub enum IndicesScope {
    Subwords,
    WordAndSubword,
```

Looks far more readable at various call sites.

Reply:
Yep, looks nice!